### PR TITLE
feat(home-manager): init delta module

### PR DESCRIPTION
**Pull request by Anomalocaridid**
Adds support for delta.

### CODE REVIEW

File: test.nix [hunk 1]
```diff
@@ -57,6 +57,7 @@ in
         bottom = ctpEnable;
         btop = ctpEnable;
         fish = ctpEnable;
+        git.delta = ctpEnable;
```
Comment (getchoo) [inline test.nix:60]: git also needs to be enabled for this to be applied - otherwise this works great. thanks!

**Review by getchoo — CHANGES_REQUESTED** [review]

File: test.nix (hunk 1 above)
Comment (Anomalocaridid) [reply test.nix:61]: Git is now enabled in test.nix

**Comment by Anomalocaridid** [comment]
I also had to update the delta source and slightly change how the theme is applied due to some upstream changes

**Review by isabelroses — APPROVED** [review]

**Review by getchoo — APPROVED** [review]

**Comment by getchoo** [comment]
thanks!

**Pull request merged**